package.json

```diff
@@ -91,8 +91,8 @@
   "xo": {
     "esnext": true,
     "ignores": [
-      "test/test.js",
-      "dist/**"
+      "dist/**",
+      "lib/index.js"
```

Comment:
Had to ignore index.js because of https://github.com/babel/babel-eslint/issues/78

It's in the known issues list: https://github.com/babel/babel-eslint#known-issues